src/simple_http.rs

```diff
@@ -155,31 +202,43 @@ impl SimpleHttpTransport {
         let mut reader = BufReader::new(sock);
 
         // Parse first HTTP response header line
-        let http_response = get_line(&mut reader, request_deadline)?;
-        if http_response.len() < 12 || !http_response.starts_with("HTTP/1.1 ") {
-            return Err(Error::HttpParseError);
+        let http_response = get_line(&mut reader, Instant::now() + self.timeout)?;
```

Comment:
What's the reason not to use `request_deadline` here? If there is one can be have a code comment please?

Reply:
I think I meant to change `request_deadline` everywhere, and it's a rebasing mistake that it hasn't been changed.

Reply:
:)